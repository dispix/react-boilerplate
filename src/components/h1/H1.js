import React from 'react'

import './H1.scss'

const H1 = ({ children }) =>
  <h1>{ children }</h1>

export default H1
