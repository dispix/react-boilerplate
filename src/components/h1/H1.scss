h1 {
  font-family: Arial;
  background-color: rgb(56, 122, 233);
  color: white;

  text-align: center;
  padding: 1em;
  border-radius: 2px;
}
