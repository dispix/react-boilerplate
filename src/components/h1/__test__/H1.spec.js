import React from 'react'
import renderer from 'react-test-renderer'

import H1 from '../H1'

test('App component snapshot', () => {
  const component = renderer.create(<H1>Test</H1>)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
