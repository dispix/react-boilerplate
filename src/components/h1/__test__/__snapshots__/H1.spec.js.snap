exports[`test App component snapshot 1`] = `
<h1>
  Test
</h1>
`;
